### PR TITLE
Supplement to local development docs

### PR DESCRIPTION
**Pull request by reginapizza**
**What type of PR is this?**
 /kind documentation

**What does this PR do / why we need it**:
Adds more complete documentation for local development steps (how to run console locally)

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [x] Documentation has been updated.

### CODE REVIEW

File: docs/devconsole/development.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 ## Contribution and developing locally
 
-1.For allowing our local UI development environment to connect to the gitops backend, we need to expose our backend using a route. Create a cluster route file `cluster-route.yaml` and add the following route to it
+1. To set up a local development environment for OpenShift Console, first clone the [console repo](https://github.com/openshift/console) and `cd` into that directory. Log into an OpenShift cluster and then run the following steps:
+
```
Comment (saumeya) [inline docs/devconsole/development.md:4]: Can `Log in` be a separate step?

**Review by saumeya — APPROVED** [review]

**Comment by saumeya** [comment]
LGTM

**Review by chetan-rns — APPROVED** [review]
Thanks @reginapizza 

/lgtm
/approve

**Pull request merged**